From 89d01c470dda92f5919b593d69b4f177489c6115 Mon Sep 17 00:00:00 2001
From: William Wu <william.wu@rock-chips.com>
Date: Tue, 2 Jun 2020 15:30:05 +0800
Subject: [PATCH] dt-bindings: usb: dwc3: add a property to do warm port reset
 quirk

This patch adds a new property "snps,xhci-warm-reset-on-suspend-quirk"
to do a warm port reset for xHC USB3 port upon suspend if needed.

Change-Id: Id26ab9ecd933f16a491b6fc13c45346459f78109
Signed-off-by: William Wu <william.wu@rock-chips.com>
---
 Documentation/devicetree/bindings/usb/dwc3.txt | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/Documentation/devicetree/bindings/usb/dwc3.txt b/Documentation/devicetree/bindings/usb/dwc3.txt
index bd831f6f371f..b9120b5eb83e 100644
--- a/Documentation/devicetree/bindings/usb/dwc3.txt
+++ b/Documentation/devicetree/bindings/usb/dwc3.txt
@@ -82,6 +82,8 @@ Optional properties:
 			register is '1').
  - snps,xhci-trb-ent-quirk: when set, the xHC use the Evaluate Next TRB(ENT)
 			flag to force the xHC to pre-fetch the next TRB of a TD.
+ - snps,xhci-warm-reset-on-suspend-quirk: when set, do a warm port reset for
+			xHC USB3 port upon suspend.
  - snps,is-utmi-l1-suspend: true when DWC3 asserts output signal
 			utmi_l1_suspend_n, false when asserts utmi_sleep_n
  - snps,hird-threshold: HIRD threshold
-- 
2.35.3

